CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 🐞 Fixed
 
 ### ⬆️ Improved
+- `ProgressTrackerFactory` and `ProgressTracker` is no more used and the attachment uploaded state is updated instead.
```

Comment:
I'd maybe rephrase this instead, it's a bit hard to understand what's going on here.

Something like:

- Removed/deprecated 'ProgressTrackerFactory` and `ProgressTracker`
- Added `UploadState` to `Attachment` for easier access when uploading items

Reply:
UploadState was already part of Attachment and was used but it wasn't used to update progress.

Reply:
Something like...

> `Attachment#uploadState` is now updated in real-time during uploads.

?

Reply:
@amitkma What I'm trying to say is that even though I know what this PR is for I still don't understand what this changelog comment is saying - it's somewhat confusing.

Something like Marton mentioned would be nicer for users, as they'd know what changed and what to use next more easily.